DESIGN.md

```diff
@@ -42,7 +42,39 @@ Consumer requests data when it wishes, and the data is then pushed when the prod
 
 ##### Observable
 
-... under discussion ... (related to Observable/Flowable debate)
+Stream that supports async and synchronous push. It does not support interactive flow control (`request(n)`).
+
+Usable for:
+
+- hot and cold sources
+- sync or async
+- push
+- 0, 1, many or infinite items
+
+Flow control support:
+
+- buffering, sampling, throttling, windowing, dropping, etc
```

Comment:
Note that sampling by itself still can overflow a client if said client doesn't request fast enough or big enough. Same is true for throttling and non-count based buffering (such as with boundary Publisher). Window is a bit both since the outer sequence may overflow the client but the inners can't because UnicastSubject buffers values (see subscription gap problem) and replays them with as the client requests.